transformations/gender_neutral_rewrite/myconstants.py

```diff
@@ -0,0 +1,66 @@
+# pronouns
```

Comment:
Thanks for the suggestion. We certainly would encourage the addition of more occupational words and gender-neutral inflections. We chose to use a `.py` file for the ability to add comments where necessary, since it may not always be immediately obvious why a certain mapping is included. We hope that users can easily add gender or occupation-specific words by following the existing template (perhaps adding an instruction to the README may not be a bad idea as well).